static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -36,9 +38,9 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/updater` - this file is used store latest available version of dvc,
   which is used to remind user to upgrade.
 
-- `.dvc/updater.lock` - a lock file for `.dvc/updater`.
+- `.dvc/updater.lock` - lock file for `.dvc/updater`
 
-- `.dvc/lock` - a lock file for the whole dvc project.
+- `.dvc/lock` - lock file for the whole dvc project
```

Comment:
Thanks! I think we should change these to `:` instead of `-` and start in capital letter for consistency, now that I think about it. But I'll do it myself, don't worry about it.